### ISSUE TITLE
This list keeps whitelisting a lot of sites...

### ISSUE THREAD

**Issue by normanhendrich** [ryanbr/fanboy-adblock#12]
Hi, 

I probably misunderstand the "green color" in the adlbock plus filter window.
But it seems that (for whatever crazy reason), this filter abonnement keeps
whitelisting a lot of things, including googletagmanager etc. again and again.

For some other reason, the abonnement overrides my own rules, but that
is an adblock plus issue, not this particular list.

Having this list does the exact opposite of what I wanted...

Cheers,

**Comment by SMed79** [comment]
Hi, 
in most cases `googletagmanager.com` is whitelisted to fix broken sites.
see → https://forums.lanik.us/search.php?keywords=googletagmanager.com&fid[0]=64

**Comment by normanhendrich** [comment]
Zitat von Med <notifications@github.com>:

> Hi,
> in most cases `googletagmanager.com` is whitelisted to fix broken sites.
> see ?   
> https://forums.lanik.us/search.php?keywords=googletagmanager.com&fid[0]=64
>
> ---
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/ryanbr/fanboy-adblock/issues/12#issuecomment-216062193

Yes, I know. But...

I use adblock to remove ads and tracking, not to keep
broken websites alive, that somebody may like or need.

The problem here is that the list-abonnement and your
whitelist OVERRIDES my custom black-list of googletagmanager.
And, there is no warning about my custom rules (which
took extra effort) simply being deleted.

How can I fix this? Everytime I restart my iceweasel,
the presumed-blocked websites are alive again. That
completely nullifies any effort you guys are putting in.

Cheers,
   Norman

**Comment by ryanbr** [comment]
Its only whitelisted on sites where its obviously broken and no way around it. You can't win, people will complain either way, its blocking too much or not blocking enough.

**Comment by normanhendrich** [comment]
Am 05.06.2016 01:51, schrieb Fanboynz:
> Its only whitelisted on sites where its obviously broken and no way
> around it. You can't win, people will complain either way, its blocking
> too much or not blocking enough.

Fair enough.

The point here is that my adblock list abonnement
kept OVERRIDING the additional filters that I added.

If a user takes the extra effort to add additional
filters beyond those included in the list abo,
those "personal" filters should always have precedence
over whatever the list considers appropriate for the
"standard" user.

Problem is, I never figured out how to keep my own
filters. When reloading the abo on start-up, the
list filters were active, and my personal filters
were overridden. (This is just the behaviour I am
used to from Windows: "we know better what is good
for you, so we have just modified your preferences").

If this is a "dumb user didn't read the manual"
problem, sorry about this. I couldn't figure out
(and Google and Bing didn't help me) to make my
own filters have precedence over the list.

Thanks again,
   Norman
